terms.html

```diff
@@ -218,6 +218,18 @@
 An identifier, as defined by [[RFC3986]].
   </dd>
 
+  <dt><dfn data-lt="verifiable timestamp">verifiable timestamp</dfn></dt>
+
+  <dd>
+A timestamp is verifiable if a third-party can verify that a piece of data and
+it's integrity exists at a moment in time.  Verifying data integrity should be
```

Comment:
```suggestion
A timestamp is verifiable if a third-party can verify that a piece of data
exists at a moment in time and has not been modified since it was 
timestamped.  Verifying data integrity should be
```